corehq/apps/campaign/models.py

```diff
@@ -1 +1,11 @@
-# Create your models here.
+from django.db import models
+
+
+class Dashboard(models.Model):
+    """
+    Model to store campaign dashboard configuration
+    """
+    domain = models.CharField(max_length=126, db_index=True, unique=True)
```

Comment:
do we need `null=False, blank=False`?

Reply:
Those are the defaults for all `Field` subclasses.